CONTRIBUTING.md

```diff
@@ -51,7 +51,16 @@ The format is a gzipped JSON `genesis.json` file, with either:
 - a `stateHash` attribute: to omit a large state (e.g. for networks with a re-genesis or migration history).
   Nodes can load the genesis block header, and state-sync to complete the node initialization.
 
-### 4. Raise your Pull Request
+### 4. Run tests locally
+Run the following command to run the registry's validation checks, for only the chain you added:
+
+```
+cd validation
+go test ./... -v -focus-chain-id=<your-chain-id-here>
```

Comment:
Could we consider ensuring that all per-chain tests use the test name & use the native `-run=regex` or does that not meet all of our requirements for this flag?

Reply:
Good suggestion. I think that could work, and would probably require less boiler plate code and a similar amount of diligence writing new tests. And it may allow focussing by name as well as by chain id, which would be nice.